src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/http.post.exchange/client.rpt

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newTargetInputRef ${nuklei:newReferenceId()} # external scope
+
+connect await ROUTED_INPUT
+        "nukleus://target/streams/http2"
+        option nukleus:route ${newTargetInputRef}
+        option nukleus:window 8192
+        option nukleus:transmission "duplex"
+
+
+write nukleus:begin.ext [0x00] [0x07] ":method" [0x04] "POST"
+                        [0x00] [0x07] ":scheme" [0x04] "http"
+                        [0x00] [0x05] ":path" [0x01] "/"
+                        [0x00] [0x0a] ":authority" [0x0e] "localhost:8080"
+
+read nukleus:begin.ext [0x00] [0x07] ":status" [0x03] "200"
+                       [0x00] [0x06] "server" [0x14] "CERN/3.0 libwww/2.17"
+                       [0x00] [0x04] "date" [0x1d] "Wed, 01 Feb 2017 19:12:46 GMT"
+                       [0x00] [0x0c] "content-type" [0x18] "text/html; charset=UTF-8"
+                       [0x00] [0x0e] "content-length" [0x03] "113"
+
+connected
```

Comment:
This script looks very weird because the connected is waiting on reading the status 200 before the POST has sent the data